http-api/src/main/java/bisq/http_api/rest_api/domain/payment_accounts/PaymentAccountsRestApi.java

```diff
@@ -169,6 +172,65 @@ public void removeAccount(@PathParam("accountName") String accountName, @Suspend
         }
     }
 
+    @PUT
+    @Operation(
+            summary = "Save payment account",
+            description = "Update an existing payment account by replacing it with new data (only UserDefinedFiatAccount)",
+            requestBody = @RequestBody(
+                    description = "Account name and new account data",
+                    required = true,
+                    content = @Content(schema = @Schema(implementation = SaveAccountRequest.class))
+            ),
+            responses = {
+                    @ApiResponse(responseCode = "204", description = "Payment account saved successfully"),
+                    @ApiResponse(responseCode = "400", description = "Invalid input data or account not found"),
+                    @ApiResponse(responseCode = "404", description = "Payment account not found"),
+                    @ApiResponse(responseCode = "500", description = "Internal server error"),
+                    @ApiResponse(responseCode = "503", description = "Request timed out")
+            }
+    )
+    @Path("/{accountName}")
+    public void saveAccount(@PathParam("accountName") String accountName,
```

Comment:
As this is for UserDefinedFiatAccount only, I think its better to reflect that in the endpoint names.